internals/__init__.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+
+# ACTIONS
+# Format: (energy change, fulfillment change, repetitive effect)
+
+
+class Action(object):
+    """Class for an action that can be done by the user.
+
+    """
+
+    def __init__(self, delta_energy, delta_fulfillment):
+        super(Action, self).__init__()
+        self.delta_energy = delta_energy
+        self.delta_fulfillment = delta_fulfillment
+
+
+ACTIONS = {
+    "drink_beer": Action(-10, +10),  # TODO: drunk_function?
+    "move_room": Action(-5, 0),  # TODO: decrease fulfillment multiplicatively
+    "eat_delivery": Action(
+        +5, +5
+    ),  # TODO: decrease energy and fulfillment multiplicatively
+    "eat_homecooked": Action(
+        +5, +10
+    ),  # TODO: decrease energy from eating too much, increase fulfillment multiplicatively
+    "screen_time": Action(
+        -5, -5
+    ),  # TODO: decrease energy, decrease fulfillment multiplicatively
+    "check_email": Action(0, 0),  # TODO: decrease fulfillment multiplicatively
+    "buy_online": Action(+10, +20),  # TODO: big decrease in energy and fulfillment
+    "binge_netflix": Action(-10, +20),  # TODO: big decrease in fulfillment
+    "cook_food": Action(-20, +20),  # TODO: big increase in fulfillment
+    "workout": Action(-20, +5),  # TODO: Fibonacci increase in fulfillment
+    "nap": Action(
+        +12, -10
+    ),  # TODO: drop fulfillment to zero if a portion of day is spent napping
+    "zoom_call": Action(-10, 0),
+    # TODO: decrease fulfillment multiplicatively
+    "people_watch": Action(0, +15),
+    "drink_caffeine": Action(+20, 0),
+    # TODO: drink too much, can't sleep/nap for 3 actions
+    "listen_to_radio": Action(0, +15),
+}
+
+
+# TIME OF DAY
+# Dictionary of day-portion tuples
+TIME_OF_DAY = {
+    "morning": ("dawn", "mid-morning"),
+    "afternoon": ("noon", "mid-afternoon"),
+    "night": ("6 pm", "8 pm", "10 pm", "midnight"),
```

Comment:
The times seem kind of out of place here, maybe you could say something like `evening` or `early/late evening`